annotations/README.md

```diff
@@ -138,7 +138,7 @@ The list identified by "short-long" contains a correspondence of short and long
 
 ## How to Annotate a Command
 
-The first step to annotating a command is to identify its default class: `stateless`, `pure`, `non-parallelizable`, and `side-effectful`. How does the command behave without any inputs?
+The first step to annotating a command is to identify its default class: `stateless`, `parallelizable_pure`, `non-parallelizable`, and `side-effectful`. How does the command behave without any inputs?
```

Comment:
As you mentioned, we should change `non-parallelizable` here to `pure` (or the opposite).